frontend/src/pages/create_user.tsx

```diff
@@ -4,13 +4,16 @@ import { useRouter } from "next/router";
 import { ReactElement, useState } from "react";
 import { FieldValues, SubmitHandler, useForm } from "react-hook-form";
 
+import { checkEmailExists } from "../firebase/firebase";
+
 import Landing from "@/components/Landing";
 import { Textfield } from "@/components/Textfield";
 import { useRedirectToHomeIfSignedIn } from "@/hooks/redirect";
 import { useWindowSize } from "@/hooks/useWindowSize";
 import { cn } from "@/lib/utils";
 
 export default function CreateUser() {
+  console.log("In create_user.tsx");
   useRedirectToHomeIfSignedIn();
```

Comment:
No need to redirect to home if signed in on create user. Login and Create User are the two flows in this app that are not auth protected.